source/committers/__index.md

```diff
@@ -8,8 +8,7 @@ There are a number of useful Apache tools and services available for Apache proj
 
 ## Apache Infrastructure Information
 
-The primary portal for Apache Infra questions - how Subversion/Git/Websites, etc. work
- at the ASF - is the [http://www.apache.org/dev/][1] pages.
+The primary portals for Apache Infra questions - how Subversion/Git/Websites, etc. work at the ASF - are pages at <a href="https://www.apache.org/dev/" target="_blank">apache.org/dev</a> and <a href="https://www.infra.apache.org" target="_blank">infra.apache.org</a>.
```

Comment:
Any reason for the target=_blank? Otherwise we could include the link in markdown style.

Reply:
I use target="_blank" to open the linked site in a new tab rather than taking over the current tab when the link takes one away from the site they intended to be on. There is a way to do it in markdown, but it is funky.

Reply:
Yeah, the markdown way of the target blank is not the best either...